examples/mugen/retrieval/model.py

```diff
@@ -106,15 +106,6 @@ def training_step(self, batch, batch_idx):
         )
         return {"loss": loss, "model_output": model_output}
 
-    def training_epoch_end(self, outputs):
```

Comment:
Just to confirm: this is why previously recall was computed at every epoch?

Reply:
correct, that function is why recall was computed for the entire training set at every epoch (recall is still computed at every epoch for the validation set in `validation_epoch_end`)